### DISCUSSION TITLE
web app example

### DISCUSSION THREAD

**Discussion by jdivanov77** [python-telegram-bot/python-telegram-bot#3176] [Q&A] [answered]
Dear friends, I am working with v13.13 and I am trying to receive data from WebView event. I have 
dispatcher.add_handler(MessageHandler(Filters.status_update.web_app_data, web_app_data))
but no data coming from this filter.
Please help! What am I missing?

I am using the example in the Wiki webappbot.py
but this example is for v20 and I am not sure is there something different for v13.13 receiving WebView data

according to v13.13 I changed

```python
application.add_handler(MessageHandler(filters.StatusUpdate.WEB_APP_DATA, web_app_data))
```

to:

```python
dispatcher.add_handler(MessageHandler(Filters.status_update.web_app_data, web_app_data))
```

and:

```python
async def web_app_data(update: Update, context: ContextTypes.DEFAULT_TYPE) -> None:
```

to:

```python
def web_app_data(update: Update, context: CallbackContext) -> None:
```

I think the other part is the same

**Comment by Bibo-Joshi** [comment, marked as answer]
The same question was asked at https://t.me/pythontelegrambotgroup/614194?thread=614189 and an "works for me" example was presented by Poolitzer at https://t.me/pythontelegrambotgroup/614831?thread=614189. Let's continue there :)
